build.gradle

```diff
@@ -30,7 +30,7 @@ plugins {
 
 allprojects {
     group 'org.radarcns'
-    version '0.3.2-SNAPSHOT'
+    version '0.3.1'
```

Comment:
why are we falling back with the earlier version?

Reply:
Not sure how this happened, it's a mistake indeed, good catch!